perun-core/src/main/java/cz/metacentrum/perun/core/api/MembersManager.java

```diff
@@ -1420,4 +1420,15 @@
 	 * @throws UserNotExistsException if there is no such user
 	 */
 	void updateSponsorshipValidity(PerunSession sess, Member sponsoredMember, User sponsor, LocalDate newValidity) throws PrivilegeException, SponsorshipDoesNotExistException, MemberNotExistsException, UserNotExistsException;
+
+
+	/**
+	 * Returns rich members with specific attributes by their IDs.
+	 *
+	 * @param sess Perun session
+	 * @param ids List<Integer> list of members IDs
+	 * @param attrsNames List<String> Attribute names
```

Comment:
Add here what happens if I send an empty list or null instead of this parameter. The same for the RPC and the open API if it is not already there.

Reply:
Fixed